sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveQuerySuite.scala

```diff
@@ -324,20 +324,6 @@ class HiveQuerySuite extends HiveComparisonTest with BeforeAndAfter {
       | FROM src LIMIT 1
     """.stripMargin)
 
-  createQueryTest("Date comparison test 2",
-    "SELECT CAST(CAST(0 AS timestamp) AS date) > CAST(0 AS timestamp) FROM src LIMIT 1")
```

Comment:
```
  override def beforeAll() {
    ...
    TimeZone.setDefault(TimeZone.getTimeZone("America/Los_Angeles"))
```

- before my fix, both Spark (1.4) and Hive (0.13) returned `true`
- after my fix, Spark (1.5) returned `false` and Hive (0.13) returned `true` (that's why we removed it)
- since Spark (1.5) later moved to Hive 1.2.1 now, both Spark and Hive return `false`